### DISCUSSION TITLE
Clickpacks

### DISCUSSION THREAD

**Discussion by TheSilentor** [maxnut/GDmenu#231] [Q&A]
Does anyone know of any clickpack(s) that I can use in GDMO? I've tried to find some but they don't work.

**Comment by Stronkkey** [comment]
Download old version and rip the clickpacks from there.

**Reply by TheSilentor** [reply]
I tried to and I can't find the clickpack folder

**Comment by 4texas4** [comment]
Where do you get clickpacks in the first place?
